__mocks__/api.js

```diff
@@ -0,0 +1,118 @@
+import { getCount } from '../src/api.js';
+
+async function fetchCount() {
```

Comment:
- [ ] Please avoid using the `function` keyword, you should use ES6 syntax so instead of `function` keyword please use `arrow functions` 😄 
    - Using ES6 syntax is a requirement for this project, kindly fix this issue. 🚀 
    - Kindly take a look at the rest of the files and change this keyword for arrow functions.
    - Using ES6 syntax make your project gives it readability along with other advantages, if you want to know more about why it is a good idea to use ES6 syntax please go to this [link](https://www.freecodecamp.org/news/write-less-do-more-with-javascript-es6-5fd4a8e50ee2/)